main.go

```diff
@@ -292,10 +298,22 @@ func handleTab() error {
         // if you successfully get an input string, grab the last word from the string
         ss := strings.Split(inputString, " ")
         s := ss[len(ss)-1]
-        // now call get the list of all possible cantidates that have that as a prefix
-        resultSlice := generateTabCompletionSlice(s)
+        // create a variable in which to store the result
+        var resultSlice []string
+        // if the word starts with a : its an emoji lookup
+        if strings.HasPrefix(s, ":") {
+            resultSlice = generateEmojiTabCompletionSlice(s)
+        } else {
+            // now in case the word (s) is a mention @something, lets remove it to normalize
+            if strings.HasPrefix(s, "@") {
+                s = strings.Replace(s, "@", "", 1)
+            }
+            // now call get the list of all possible cantidates that have that as a prefix
+            resultSlice = generateChannelTabCompletionSlice(s)
+        }
         lcp := longestCommonPrefix(resultSlice)
         if lcp != "" {
+            printToView("Feed", fmt.Sprintf("%d tab completion options", len(resultSlice)))
```

Comment:
At some point it would be nice if this showed up to 5 emojis that are being found, and then finish with `and X more` if there are more than 5.

Reply:
Yes this was the original goal, for now its just debug output.  There were so many emoji options that i wasn't getting good results and before i dumped them all to screen i wnated to know how many options i was getting.  Were there 8? or 90?